### PR TITLE
Add RUM integration for React

### PR DESCRIPTION
**Pull request by milena-markovic**
### What does this PR do?

Adds Datadog RUM integration tile for React apps.

### Motivation

Surfacing the existing React-RUM integration

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

- [x] Logo alias ready in druids

### CODE REVIEW

**Comment by hestonhoffman** [comment]
I've added an [editorial review card for the docs team](https://datadoghq.atlassian.net/browse/DOCS-3722)

**Review by hithwen — APPROVED** [review]

File: rum_react/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# React Integration
+
+## Overview
+
+WWith the Datadog [React integration][1], resolve performance issues quickly in React components by:
```
Comment (hestonhoffman) [inline rum_react/README.md:5]:
```suggestion
With the Datadog [React integration][1], resolve performance issues quickly in React components by:
```

File: rum_react/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- Debugging the root cause of performance bottlenecks, such as a slow server response time, render-blocking resource, or error inside a component
```
Comment (hestonhoffman) [inline rum_react/README.md:7]:
```suggestion
- Debugging the root cause of performance bottlenecks, such as a slow server response time, render-blocking resource, or an error inside a component
```

File: rum_react/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### Collect traces 
+
+Your React application automatically sends traces to Datadog.
+
+### Collect logs 
+
+To start forwarding your React application's logs to Datadog, see [React Log Collection][3].
+
+Data Collected
```
Comment (hestonhoffman) [inline rum_react/README.md:33]:
```suggestion
## Data Collected
```

File: rum_react/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+Need help? Contact [Datadog Support][6]. 
+
+## Further Reading 
+
+Additional helpful documentation, links, and articles: 
+
+- [React Monitoring][7]
+
+
+## Links
```
Comment (hestonhoffman) [inline rum_react/README.md:58]:
```suggestion
```

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**